plenum/test/consensus/test_consensus_data_provider.py

```diff
@@ -23,5 +27,33 @@ def test_initial_consensus_state(some_item, other_item, validators):
     assert data.prepared == []
 
     # Checkpoints
-    assert data.stable_checkpoint == 0
-    assert data.checkpoints == []
+    assert data.stable_checkpoint == None
+    assert list(data.checkpoints) == []
+
+
+def test_pp_storages_ordering(pre_prepare, consensus_data_helper: ConsensusDataHelper):
+    consensus_data_helper.preprepare_batch(pre_prepare)
```

Comment:
I think it's better to move these tests which use ConsensusDataHelper to new file `test_consensus_data_helper`, which probably should belong to replica tests directory.

Reply:
ok